locale/fa_IR/countries.xml

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE countries SYSTEM "../../dtd/countries.dtd">
+
+<!--
+  * lib/pkp/locale/fa_IR/countries.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
Copyrights should be to Simon Fraser University, not Simon Fraser University Library.

Reply:
All copyright dates for Simon Fraser University and John Willinsky should end -2018.